src/gluonts/meta/style.py

```diff
@@ -0,0 +1,72 @@
+from pygments.style import Style
+from pygments.token import (
+    Comment,
+    Keyword,
+    Name,
+    String,
+    Number,
+    Operator,
+    Whitespace,
+)
+
+from . import colors
+
+
+# class Light(Style):
+#     background_color = "rgba(0, 0, 0, 0.05)"
+
+#     styles = {
+#         Comment: f"italic #888",
+#         Comment.Preproc: f"noitalic {colors.GREEN}",
```

Comment:
Is this meant to go? If so, maybe we could remove it

Reply:
I wanted to also have a light style, however the colors don't work so well. I guess I can just remove this anyway and revisit it later.